pkgs/games/dwarf-fortress/dfhack/default.nix

```diff
@@ -148,29 +155,69 @@ in
     # functionality.
     postPatch = ''
       sed -i 's@cached_path = path_string.*@cached_path = getenv("DF_DIR");@' library/Process-linux.cpp
+
+      # shadows something in cmath.h
+      sed -i 's@lerp@_lerp@g' plugins/remotefortressreader/remotefortressreader.cpp
```

Comment:
We should use substituteInPlace --replace-fail if possible to notice silent breakages